go.mod

```diff
@@ -42,3 +42,5 @@ require (
 	golang.org/x/sys v0.0.0-20220114195835-da31bd327af9
 	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1
 )
+
+replace github.com/containers/ocicrypt => github.com/containers/ocicrypt v1.1.4-0.20220428134531-566b808bdf6f
```

Comment:
Is this line really necessary? I think just bumping the version above should be enough. (And it kind of _has_ to be enough because `replace` directives only work at the top level, i.e. this wouldn’t apply to a Skopeo/Podman build.)

Reply:
`go mod download` didn't let me fetch `v1.1.4` so I kept the line with the version-commit as-is and got rid of the replace directive. PTAL.

Reply:
`go mod download` isn’t supposed to update `go.mod` I think…

FWIW `go get github.com/containers/ocicrypt@v1.1.4` worked for me, resulting in
```
github.com/containers/ocicrypt v1.1.4
```
. I didn’t realize this pre-release version actually points at a tagged version — in that case, I think using the 1.1.4 tag would be a bit safer WRT any other consumers of that.